mw-webapp/src/pages/authPage/AuthPage.tsx

```diff
@@ -16,7 +16,7 @@ export const AuthPage = () => {
   const [user, setUser] = useState<User | null>(null);
 
   useEffect(() => {
-    handleUserAuthState(setUser);
+    userAuthState(setUser);
     writeNewUserCredentials();
```

Comment:
writeNewUserCredentials calls on mounting AuthPage and it set data to user:
`
ownWays: [""],
favoriteWays: [""],
mentoringWays: [""],
` 
Looks like it will reset all my favorite ways every time I'm logged in, it will be so sad for users)
What do you think?

Reply:
Yes, you are right.
I will set only the name, email, and uid for now. Later we can write logic to check if the user uid is already in Firebase and then keep his ways.
Maybe don't get a user name from Google also? If the user can change his name on our site then when the user login again, the name will also reset. What do you think?

Reply:
Yeah, it looks better! Also we could check firebase docs, maybe they provide appropriate method.

Reply:
Move adding a new user to DB logic and checking if a user already exists to  handleUserAuthState function